RELEASE_NOTES.md

```diff
@@ -1,5 +1,77 @@
 <!-- markdown-link-check-disable -->
 
+# 0.70.0
+
+The **ZenML 0.70.0** release includes a significant number of database schema changes and migrations, which means upgrading to this version will require extra caution.
+
+## Key Changes
+
+* **Artifact Versioning Improvements**: The handling of artifact versions has been improved, including the ability to batch artifact version requests and more flexible step inputs/outputs management and versioning.
```

Comment:
Can we link to some docs as to what this means?

Reply:
This is kinda hidden and was not documented, but it will be visible on the dashboard.

Reply:
maybe a better explaianation then @avishniakov ?

Reply:
Improved, thanks for noticing this